### ISSUE TITLE
Bug fix: In deployment code, z and v are in wrong order (latent and vel reversed)

### ISSUE THREAD

**Issue by antares310** [ShengqianChen/Go2W_DreamWaQ#4]
I found a critical bug in the deployment script.

In training code:
observations = torch.cat((z, v, observations), dim=-1)

In deployment code:
code = torch.cat((vel, latent), dim=-1)
The order is wrong (vel/v first, latent/z second).

This causes the robot dog to be unstable after deployment.

Fix: swap latent and vel:
code = torch.cat((latent, vel), dim=-1)

**Comment by ShengqianChen** [comment]
Thank you for the detailed report! This was indeed a critical bug --- the mismatch in feature concatenation order between training and deployment causes a distribution shift in policy inputs, leading to unstable behavior after deployment.

We have fixed this in commit \texttt{fix: correct policy code feature concatenation order}, aligning the deployment code with the training code.

Thanks again for catching this --- it really helps improve the reliability of the codebase. Please let us know if you run into any other issues after deployment!
